### PR TITLE
Moving away from the Rx naming scheme

### PR DESCRIPTION
**Pull request by xvzf**
@JonasVautherin @julianoes  FYI, please merge asap

### CODE REVIEW

File: pb_plugins/dcsdkgen.egg-info/PKG-INFO [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
+Name: dcsdkgen
+Version: 0.1a0
+Summary: Autogenerator for the DroneCore SDK Bindings
```
Comment (julianoes) [inline pb_plugins/dcsdkgen.egg-info/PKG-INFO]: `DroneCore` -> `Dronecode`

File: pb_plugins/dcsdkgen.egg-info/PKG-INFO [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Home-page: @TODO
+Author: @TODO
```
Comment (julianoes) [inline pb_plugins/dcsdkgen.egg-info/PKG-INFO]: You? :smile:

File: pb_plugins/dcsdkgen.egg-info/PKG-INFO [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+Author-email: @TODO
+License: UNKNOWN
```
Comment (julianoes) [inline pb_plugins/dcsdkgen.egg-info/PKG-INFO]: BSD 3-clause.

File: pb_plugins/dcsdkgen/methods.py [hunk 4]
```diff
@@ -113,38 +113,38 @@ def collect_methods(
...
 
             # Check if method is observable
-            elif (is_observable(method)):
-                _methods[method.name] = ObservableMethod(plugin_name,
-                                                         package,
-                                                         template_env,
-                                                         method,
-                                                         requests,
-                                                         responses)
+            elif (is_stream(method)):
```
Comment (julianoes) [inline pb_plugins/dcsdkgen/methods.py]: Comment above outdated.

File: pb_plugins/dcsdkgen/methods.py [hunk 5]
```diff
@@ -113,38 +113,38 @@ def collect_methods(
...
 
         return _methods
 
     def __repr__(self):
         return "method: {}".format(self._name)
 
 
-class CompletableMethod(Method):
+class Call(Method):
     """ A completable method doesn't return any value,
```
Comment (julianoes) [inline pb_plugins/dcsdkgen/methods.py]: Comment...

Comment (xvzf) [reply pb_plugins/dcsdkgen/methods.py]: I'll run an sed over it, there is so much dronecore still in it :-D

File: pb_plugins/dcsdkgen.egg-info/PKG-INFO (hunk 3 above)
Comment (xvzf) [reply pb_plugins/dcsdkgen.egg-info/PKG-INFO]: File should not be in git, added the whole folder to gitignore

**Comment by xvzf** [comment]
@julianoes Naming and comments should be fixed now, please have a look!

**Pull request merged**